director/auth/test_api_views.py

```diff
@@ -0,0 +1,131 @@
+import time
+
+from _pytest.monkeypatch import MonkeyPatch
+from django.contrib.auth.models import User
+from django.urls import reverse
+from rest_framework import status
+from rest_framework_jwt.serializers import jwt_payload_handler, jwt_encode_handler
+from rest_framework.test import APITestCase
+import google.oauth2.id_token
+import jwt
+
+from auth.api_views import OpenIdGrantView, GOOGLE_ISS, GOOGLE_AUDS
+
+
+class OpenIdGrantViewTests(APITestCase):
+
+    url = reverse("api_openid_grant")
+
+    def setUp(self):
+        self.monkeypatch = MonkeyPatch()
+
+    def post_token(self, claims):
+        """Make a request payload with token"""
+        return self.client.post(self.url, {"token": jwt_encode_handler(claims)})
+
+    def mock_google_verify(self):
```

Comment:
I have been using unittest.mock throughout other tests to achieve these kind of things.

Reply:
Thanks, yep that's a better alternative. Changed.